packages/main/test/samples/TabContainer.sample.html

```diff
@@ -267,20 +267,20 @@ <h3>TabContainer with text and additional-text</h3>
 	<h3>TabContainer with tabLayout="Inline"</h3>
 	<div class="snippet">
 		<ui5-tabcontainer tab-layout="Inline" collapsed fixed>
-			<ui5-tab text="Monitors" additional-text="(10)">
+			<ui5-tab text="Monitors" additional-text="10">
 			</ui5-tab>
-			<ui5-tab text="Cameras" additional-text="(2)" selected>
+			<ui5-tab text="Cameras" additional-text="2" selected>
 			</ui5-tab>
-			<ui5-tab text="Rooms" additional-text="(16)">
+			<ui5-tab text="Rooms" additional-text="16">
 			</ui5-tab>
 		</ui5-tabcontainer>
 
 		<ui5-tabcontainer tab-layout="Inline" collapsed fixed>
-			<ui5-tab icon="laptop" text="Monitors" additional-text="(10)">
+			<ui5-tab icon="laptop" text="Monitors" additional-text="10">
 			</ui5-tab>
-			<ui5-tab icon="video" text="Cameras" additional-text="(2)" selected>
+			<ui5-tab icon="video" text="Cameras" additional-text="2" selected>
 			</ui5-tab>
-			<ui5-tab icon="home" text="Rooms" additional-text="(16)">
+			<ui5-tab icon="home" text="Rooms" additional-text="16">
```

Comment:
You need to also update the snippets, that are displayed to the readers, the section starts few lines below with 
```html
<pre class="prettyprint lang-html"><xmp>
```